polaris-react/src/components/Tabs/Tabs.tsx

```diff
@@ -524,7 +524,12 @@ export const Tabs = ({
           });
         }
       }}
-      icon={<Icon source={PlusIcon} accessibilityLabel={createViewA11yLabel} />}
+      icon={
+        <Icon
+          source={PlusCircleIcon}
```

Comment:
cc @sarahill I had to apply this change directly to the `Tabs` component. Just want to confirm we should be using `PlusCircleIcon` in all cases?

Reply:
Ah I was wondering about that. Maybe lets hold off on that and make a separate issue. I want to get sign off before making the change to the component.